src/agent/ncp_openthread.cpp

```diff
@@ -213,6 +213,10 @@ void ControllerOpenThread::Reset(void)
     otInstanceFinalize(mInstance);
     otSysDeinit();
     Init();
+    for (auto &handler : mResetHandlers)
```

Comment:
Do we need multiple notifications?

Reply:
Just for possibility of extension if we do support multiple IPC mechanisms.